pkgs/development/python-modules/cyipopt/default.nix

```diff
@@ -0,0 +1,49 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  cython,
+  ipopt,
+  numpy,
+  pkg-config,
+  pytestCheckHook,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "cyipopt";
+  version = "1.5.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "mechmotum";
+    repo = "cyipopt";
+    rev = "v${version}";
+    hash = "sha256-ddiSCVzywlCeeVbRJg2wxKIlAVlZw9Js95IbEDqhh5Q=";
+  };
+
+  nativeBuildInputs = [
+    pkg-config
+    pytestCheckHook
+  ];
+
+  buildInputs = [ ipopt ];
+
+  build-system = [
+    cython
+    numpy
```

Comment:
```suggestion
```

Reply:
Build-time dependencies and runtime dependencies are different, so numpy must be added to each input.
https://github.com/mechmotum/cyipopt/blob/9fe5b202d51294bfcc7b0d20d560eea49b2c344b/pyproject.toml#L2

Reply:
I'm not sure why this conversation was marked as resolved, but please add `numpy` to `build-system`.